packages/css-framework/src/components/_date-picker.scss

```diff
@@ -151,42 +151,37 @@
   }
```

Comment:
There are a few places where `f.spacing` could be used here.

Reply:
Could you further clarify where else could utilise f.spacing?